### ISSUE TITLE
add `--access=public` to release script

### ISSUE THREAD

**Issue by userquin** [vite-pwa/sveltekit#11]

**Comment by th3l0g4n** [comment]
Is this related to the issue, that this package cannot be downloaded?
When I try to install `@vite-pwa/sveltekit` I get a 404 from the registry :/

**Comment by userquin** [comment]
@th3l0g4n yes and no, `@vite-pwa/sveltekit` package hasn't been released/published yet

**Comment by kwaa** [comment]
> @th3l0g4n yes and no, `@vite-pwa/sveltekit` package hasn't been released/published yet

Is there a planned release date?

**Comment by Stokoloko89** [comment]
When can we install this?

**Comment by userquin** [comment]
closing this, we need to include a few things... rn publishing manually ;)

released `v0.0.1`
